artix7/make/project_xray.cmake

```diff
@@ -1,90 +1,263 @@
-set(
-  PRJXRAY_DIR
+set(PRJXRAY_DIR ${symbiflow-arch-defs_SOURCE_DIR}/third_party/prjxray)
+set(PRJXRAY_DB_DIR
   ${symbiflow-arch-defs_SOURCE_DIR}/third_party/prjxray-db
-  CACHE PATH "Path to prjxray database files"
-)
+  CACHE PATH "Path to prjxray database files")
+
+add_conda_pip(
+  NAME progressbar2
+  NO_EXE
+  )
+
+add_conda_pip(
+  NAME simplejson
+  NO_EXE
+  )
+
+add_conda_pip(
+  NAME intervaltree
+  NO_EXE
+  )
```

Comment:
We probably should just have a `requirements.txt` file and get conda to install using that... Lets log an issue..

Reply:
#279